neuror/main.py

```diff
@@ -603,9 +602,35 @@ def _fill_repair_type_map(self):
                 self.repair_type_map[section] = RepairType.trunk
 
 
-def repair(inputfile, outputfile, axons=None, seed=0, plane=None, plot_file=None):
-    '''The repair function'''
+def repair(inputfile, outputfile, axons=None, seed=0, cut_leaves_coordinates=None, plot_file=None):
+    '''The repair function
+
+    Args:
+        inputfile: the input morph
+        outputfile: the output morph
+        axons: the axons
+        seed: the numpy seed
+        cut_leaves_coordinates: List of 3D coordinates from which to start the repair
```

Comment:
this is one of the main function of the lib. You would benefit from adding the types for the inputs